docs/user-manual/en/configuring-transports.md

```diff
@@ -499,22 +499,32 @@ for more details.
   [broker's classpath](using-server.md#adding-runtime-dependencies).
 
 
-#### Configuring a SSLContextFactory
+#### Configuring an SSLContextFactory
```

Comment:
Why 'an' ?

Reply:
Because `SSL` is an abbreviation and you speak it with the vowel "e" first, i.e. "an Es Es El"